test/test_runner.sh

```diff
@@ -21,16 +21,12 @@ fi
 # Load the helpers.
 . helpers.bash
 
-function execute() {
-    echo >&2 ++ "$@"
-    time "$@"
-}
-
 # Tests to run. Default is "." (i.e. the current directory).
 TESTS=("${@:-.}")
 
-# The number of parallel jobs to execute
+# The number of parallel jobs to execute tests
 export JOBS=${JOBS:-$(($(nproc --all) * 4))}
 
 # Run the tests.
-execute bats --jobs "$JOBS" --tap "${TESTS[@]}"
+bats --jobs "$JOBS" --tap "${TESTS[@]}"
+TEST_SERIAL="Yes" bats --tap ./irqbalance.bats
```

Comment:
does calling `execute` twice not work?

Reply:
I haven't tried but just followed [this](https://github.com/cri-o/cri-o/pull/6777#discussion_r1153819894) comment